reactioncommerce/reaction
# refactor: move payments, payments-example, payments-stripe from server/no-meteor to node-app

Resolves #5609 #5610 #5611 
Impact: **minor**  
Type: **refactor**

As part of our effort to separate our Reaction Admin from our Reaction Node API, we are moving our GraphQL services from their respective `/imports/plugins/*/server/no-meteor/` folders into their new `/node-app/{core-services|plugins}/*` folder.

Updates also include future-proofing some existing code, including adding filetype extensions (and `index.js` to imports)

## Breaking changes
Any custom plugins that import files directly from the `payments`, `payments-example`, `payments-stripe` packages will need to be updated.

## Notes
There are many places in the `orders` package that import from the payments package. I've created a ticket to address this later, and made TODO's in each file: https://github.com/reactioncommerce/reaction/issues/5653

## Testing
1. Start the application and see that it works as it should.
1. Create orders using both stripe and example payment method, see they work
